src/sentry/workflow_engine/types.py

```diff
@@ -42,7 +42,7 @@ def bulk_get_query_object(data_sources) -> dict[int, T | None]:
 
 class DataConditionHandler(Generic[T]):
     @staticmethod
-    def evaluate_value(value: T, comparison: Any, condition: str) -> DataConditionResult:
+    def evaluate_value(value: T, comparison: Any, condition: str, **kwargs) -> DataConditionResult:
```

Comment:
can you explain how these kwargs are being used?

Reply:
we discussed this before, we can pass the other parts of `PostProcessJob` that aren't the `GroupEvent` as `kwargs` so they can be evaluated in workflow conditions, and keep `process_workflows` general for other group types that don't have kwargs. there is an `EventState` object we check for RegressionCondition and ReappearedEventCondition today, for example